spec/services/food_service_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Food API' do
+  it 'returns food data' do
+    food_data = FoodService.get_details("sweet potatoes")
+
+    expect(food_data).to be_an(Array)
+    expect(food_data.first).to be_a(Hash)
+
+    sweet_potato = food_data[0]
+
+    expect(sweet_potato[:gtinUpc]).to eq("492111402857")
+    expect(sweet_potato[:foodCategory]).not_to eq("070038322238")
+    expect(sweet_potato[:brandOwner]).not_to eq(sweet_potato[:brandName])
```

Comment:
No need for these tests at the service level. This is better tested at the PORO level, which you did, so this is redundant.